src/OpenFeature.Contrib.ConfigCat/UserBuilder.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using ConfigCat.Client;
+using OpenFeature.Model;
+
+namespace OpenFeature.Contrib.ConfigCat
+{
+    internal static class UserBuilder
+    {
+        private static readonly string[] PossibleUserIds = { "ID", "IDENTIFIER" };
```

Comment:
It may be worth documenting how this mapping works in a README.

Reply:
Added a README with that info.